service/controller/v12/templates/cloudconfig/nvme_udev_hack.go

```diff
@@ -0,0 +1,26 @@
+package cloudconfig
+
+const NVMEUdevRule = `KERNEL=="nvme[0-9]*n[0-9]*", ENV{DEVTYPE}=="disk", ATTRS{model}=="Amazon Elastic Block Store", PROGRAM="/opt/ebs-nvme-mapping /dev/%k", SYMLINK+="%c"
```

Comment:
udev rule that calls script on any events with nvme that will create/delete sylinks with name that was specified in EBS e.g. `/dev/xvdh`